### PR TITLE
Clean up codes with lambda expression

### PR DESCRIPTION
**Pull request by imbyungjun**
- Use lambda expression
- Use diamond operator

### CODE REVIEW

**Comment by donggyu04** [comment]
Can one of the admins verify this patch?

File: ngrinder-controller/src/main/java/org/ngrinder/agent/service/AgentManagerService.java [hunk 1]
```diff
@@ -208,7 +203,7 @@ int getMaxAgentSizePerConsole() {
 	public List<AgentInfo> getAllLocalWithFullInfo() {
 		Map<String, AgentInfo> agentInfoMap = createLocalAgentMap();
 		Set<AgentIdentity> allAttachedAgents = getAgentManager().getAllAttachedAgents();
-		List<AgentInfo> agentList = new ArrayList<AgentInfo>(allAttachedAgents.size());
+		List<AgentInfo> agentList = new ArrayList<>(allAttachedAgents.size());
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/agent/service/AgentManagerService.java]: Can't we use stream here?

File: ngrinder-controller/src/main/java/org/ngrinder/perftest/model/NullSingleConsole.java [hunk 2]
```diff
@@ -29,7 +29,7 @@
  */
 public class NullSingleConsole extends SingleConsole {
 
-	private static final Map<String, Object> EMPTY_RESULT = new HashMap<String, Object>(1);
+	private static final Map<String, Object> EMPTY_RESULT = new HashMap<>(1);
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/perftest/model/NullSingleConsole.java]: Just use ```Maps.of()```

File: ngrinder-controller/src/main/java/org/ngrinder/security/SecuredUser.java [hunk 3]
```diff
@@ -64,7 +64,7 @@ public SecuredUser(User user, String userInfoProviderClass) {
 	 */
 	@Override
 	public Collection<GrantedAuthority> getAuthorities() {
-		List<GrantedAuthority> roles = new ArrayList<GrantedAuthority>(1);
+		List<GrantedAuthority> roles = new ArrayList<>(1);
 		roles.add(new SimpleGrantedAuthority(getUser().getRole().getShortName()));
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/security/SecuredUser.java]: return List.of(new SimpleGrantedAuthority(getUser().getRole().getShortName()))

File: ngrinder-controller/src/main/java/org/ngrinder/perftest/service/TagService.java [hunk 4]
```diff
@@ -96,7 +96,7 @@
 	 * @return found tag string lists
 	 */
 	public List<String> getAllTagStrings(User user, String query) {
-		List<String> allString = new ArrayList<String>();
+		List<String> allString = new ArrayList<>();
 		for (Tag each : getAllTags(user, query)) {
 			allString.add(each.getTagValue());
 		}
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/perftest/service/TagService.java]: Can't we use stream here?

File: ngrinder-controller/src/main/java/org/ngrinder/perftest/service/AgentManager.java [hunk 5]
```diff
@@ -434,7 +423,7 @@ public void run() {
 	 * @return selected agent.
 	 */
 	public Set<AgentIdentity> selectAgent(User user, Set<AgentIdentity> allFreeAgents, int agentCount) {
-		Set<AgentIdentity> userAgent = new HashSet<AgentIdentity>();
+		Set<AgentIdentity> userAgent = new HashSet<>();
 		for (AgentIdentity each : allFreeAgents) {
 			String region = ((AgentControllerIdentityImplementation) each).getRegion();
 			if (StringUtils.endsWith(region, "owned_" + user.getUserId())) {
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/perftest/service/AgentManager.java]: Can't we use stream here?

File: ngrinder-controller/src/main/java/org/ngrinder/perftest/service/AgentManager.java [hunk 6]
```diff
@@ -315,14 +309,14 @@ AgentControllerIdentityImplementation convert(AgentIdentity identity) {
...
 		for (AgentInfo each : cachedLocalAgentService.getLocalAgents()) {
 			if (each.isApproved()) {
 				ips.add(each.getIp() + each.getName());
 			}
 		}
-		Set<AgentIdentity> approvedAgent = new HashSet<AgentIdentity>();
+		Set<AgentIdentity> approvedAgent = new HashSet<>();
 		for (AgentIdentity each : agents) {
 			if (ips.contains(((AgentControllerIdentityImplementation) each).getIp() + each.getName())) {
 				approvedAgent.add(each);
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/perftest/service/AgentManager.java]: Can't we use stream here?

**Pull request merged**